src/sentry/tasks/reports.py

```diff
@@ -576,7 +577,7 @@ def prepare_reports(dry_run=False, *args, **kwargs):
     logger.info("reports.begin_prepare_report")
 
     organizations = _get_organization_queryset()
-    for organization in organizations:
+    for organization in RangeQuerySetWrapper(organizations, step=10000):
```

Comment:
does it matter that the queryset here has a filter? I see in our docs:

>We generally prefer to avoid using .filter with RangeQuerySetWrapperWithProgressBar. Since it already orders by the id to iterate through the table, we can't take advantage of any indexes on the fields, and could potentially scan a large number of rows for each chunk.

Reply:
In this case I think it's ok. There's no index on `status`, and we were already seq scanning the entire table with this query. This will now change to doing ~100 smaller seq scans, so the difference shouldn't be a big deal.